### PR TITLE
EVG-13300 show PR commit queue items

### PR DESCRIPTION
**Pull request by ablack12**
image

Yay! Sorry if this is super flawed.
For PRs, the first item in the queue might have a patch associated with it but for everything else we _only_ know the PR number, so the link is to the PR in github.

### CODE REVIEW

**Review by john-m-liu — CHANGES_REQUESTED** [review]
Looks good but can you add a test for a PR commit queue? You'll probably need to do the following
- add a project with a pr commit queue + its corresponding commit queue to the testdata/local data in evergreen. Can probably enable it for the logkeeper project
- add a cypress integration test here that visits the url for that commit queue and asserts that certain elements are there

File: src/gql/generated/types.ts [hunk 1]
```diff
@@ -164,7 +165,7 @@ export type Mutation = {
   setTaskPriority: Task;
   restartTask: Task;
   saveSubscription: Scalars["Boolean"];
-  removePatchFromCommitQueue?: Maybe<Scalars["String"]>;
+  RemoveItemFromCommitQueue?: Maybe<Scalars["String"]>;
```
Comment (malikchaya2) [inline src/gql/generated/types.ts:168]: once you change this to camelcase in Evergreen, you'll have to `npm run codegen` again to change it here.

**Review by malikchaya2 — COMMENTED** [review]
This looks great! Should be gtg after tests.

File: src/pages/commitqueue/CommitQueueCard.tsx [hunk 2]
```diff
@@ -5,47 +5,53 @@ import Button from "@leafygreen-ui/button";
...
+  owner,
+  repo,
   moduleCodeChanges,
   commitQueueId,
 }) => {
-  const [removePatchFromCommitQueue, { loading, error }] = useMutation<
-    RemovePatchFromCommitQueueMutation,
-    RemovePatchFromCommitQueueMutationVariables
-  >(REMOVE_PATCH_FROM_COMMIT_QUEUE);
+  const [RemoveItemFromCommitQueue, { loading, error }] = useMutation<
```
Comment (SupaJoon) [inline src/pages/commitqueue/CommitQueueCard.tsx:46]: nit: we usually make this variable camel case

**Comment by SupaJoon** [comment]
You can ignore the failing `test` and `coverage` tasks because they are flakey at the moment

**Comment by khelif96** [comment]
in order to resolve the `package-lock` merge conflict delete the file and run `npm install` .

**Comment by khelif96** [comment]
Sorry should have clarified. Merge master, delete the file, then run npm install to regenerate it.

**Comment by khelif96** [comment]
Actually on second thought the file should not have changed at all since did not make any changes to any dependencies. or else package.json would have changed too.

**Review by john-m-liu — APPROVED** [review]

**Pull request merged**